docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -824,16 +866,18 @@ quarkus.oidc.public-key=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y
 smallrye.jwt.sign.key.location=/privateKey.pem
 ----
 
-copy link:{quarkus-blob-url}/integration-tests/oidc-tenancy/src/main/resources/privateKey.pem[privateKey.pem] from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the `Wiremock` section above to generate JWT tokens. You can use your own test keys if preferred.
+To generate JWT tokens, copy `privateKey.pem` from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the preceding <<integration-testing-wiremock>> section.
+You can use your own test keys, if preferred.
 
-This approach provides a more limited coverage compared to the Wiremock approach - for example, the remote communication code is not covered.
+This approach provides limited coverage compared to the WireMock approach.
+For example, the remote communication code is not covered.
 
 [[integration-testing-security-annotation]]
 ==== TestSecurity annotation
 
-You can use `@TestSecurity` and `@OidcSecurity` annotations for testing the `service` application endpoint code which depends on the injected `JsonWebToken` as well as `UserInfo` and `OidcConfigurationMetadata`.
+You can use `@TestSecurity` and `@OidcSecurity` annotations to test the `service` application endpoint code that depends on the injected `JsonWebToken`, `UserInfo`, `OidcConfigurationMetadata`, or all three.
```

Comment:
@sberyozkin, in the OIDC  code flow authentication topic, for this section we made some changes. For consistency, shall I also update this section to the following: (but replacing `web-app` application with `service` application)?
"You can use @TestSecurity and @OidcSecurity annotations to test the `service`application endpoint code, which depends on either one, or all four, of the following injections:

Reply:
@sheilamjones, sure, please do it, (it is only 3 injections in this case)